docs/changelog.mdx

```diff
@@ -7,6 +7,33 @@ This is the changelog for the Testkube Control Plane, latest releases are on top
 - **Patch releases** focus on critical bugs and security issues.
   :::
 
+## Patch Release v1.16.0 (2025-06-PLACHOLDER)
+
+## Features
+
+Testkube Helm Chart for Runner - PLACHOLDER Section
+
+Testkube Shared views - PLACHOLDER Section
```

Comment:
I think they're called "Custom Views"? we don't have any docs lined up afaik - will need to create those..